<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QDialog" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Linguagens</string>
  </property>
  <widget class="QLabel" name="Titulo_menu">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Inserção de Linguagens</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_linguagem">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_linguagem">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Linguagem</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Criador">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_criador">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Criador</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ano">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ano</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ano">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Adicionar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_exit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>270</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_busca">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>230</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Busca</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_del">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
